src/Microsoft.AspNet.Server.Kestrel/Networking/Libuv.cs

```diff
@@ -2,413 +2,44 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Reflection;
 using System.Runtime.InteropServices;
-using System.Text;
 
 namespace Microsoft.AspNet.Server.Kestrel.Networking
 {
-    public class Libuv
+    public static class Libuv
     {
-        public Libuv()
+        public static bool IsWindows
         {
-            IsWindows = PlatformApis.IsWindows();
-            if (!IsWindows)
+            get
             {
-                IsDarwin = PlatformApis.IsDarwin();
+#if ASPNETCORE50
+                return true;
```

Comment:
While this is true now, it won't be true in the future... what would be the correct code once Core CLR runs on non-Windows?